sdk/code-analysis/AnalyzerUnitTests/Verifiers/CSharpAnalyzerVerifier`1+Test.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.CodeAnalysis.CSharp.Testing;
```

Comment:
Everything in the unit test project's `Verifiers` folder is code you get today when you create a new code analysis with the VS template. I have copied it here so for my new unit test I can take advantage of it.